docs/manual.rst

```diff
@@ -137,3 +137,44 @@ below.
       stderr:
         contains:
          - "(100%) done"
+
+WDL with Cromwell example
+-------------------------
+
+An example yaml file that could be used to test a WDL pipeline that is run with
+Cromwell is explained below.
```

Comment:
This sentence is a bit hard to follow. Maybe something like this would be better:
"Below an example yaml file is explained, which can be used to test a WDL pipeline run through Cromwell."